packages/cx-templates/doc/Generic.md

```diff
@@ -2,26 +2,31 @@
 
 ## Overview
 
-The generic template is barebones templates that provides four sections on page.
+The Generic Template is a barebones template that provides four sections to a page:
 
-* Top is full width viewport section.  This will be limited to a select group of comopnents that work well for full width and its intention from design perspective is it will be the "hero".
-* Breadcrumb which is reserved slot for site's breadcrumb if its used on that page.
-* Content section, which also is a constrain section where primary content lives.
-* Bottom section, which also is a constrain section where items such as related content or advertisements could live.
+- **Top:** A full-width viewport section limited to a select group of components that work well for
+  full-width, and its intention, from a design perspective, is that it will be the _hero_.
+- **Breadcrumb:** A reserved slot for your site's breadcrumbs (if it's used on that page).
+- **Content:** A constrained section where primary content lives.
+- **Bottom:** A constrained section where items such as related content or advertisements could
+  live.
 
-## Usage
+## Content Editor Details
 
-## Content Editor
+There is no interaction by the Content Editor with the template itself, but with the components
+available that are provided in the sections.
 
-There is no content editor's interaction with the template itself but with the components available that are provided in the sections.
+## Site Builder Details
 
-## Site Builder
+This generic template is set up in the Vital Design System to be the default template provided to
+users.
 
-This generic template is setup in Vital design system to be the default template provided to users.
+What template is provided to the user is set by tokens with the Page Component. For example, within
```

Comment:
Actually this probably doesn't explain it enough -- I maybe would start with "for how to use this generic template compoment" see -> Templates | Sit buidler section."